### ISSUE TITLE
Provide Pull Request icons for Closed and Merged

### ISSUE THREAD

**Issue by nschonni** [primer/octicons#165]
Similar to how "issues" has the closed icon with the checkmark, there should be a non-colour based way to differentiate open/closed/merged pull requests.

This would improve the accessibility for those that have trouble differentiating colours https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html

/cc @muan for the accessibility opinion

**Comment by shurcooL** [comment]
FWIW, issues not only has [opened](https://octicons.github.com/icon/issue-opened/) and [closed](https://octicons.github.com/icon/issue-closed/) icons, it also has a [reopened](https://octicons.github.com/icon/issue-reopened/) one!

On the topic of merged pull requests, there is a [git-merge](https://octicons.github.com/icon/git-merge/) icon. But it might be more related to git operations, rather than to pull requests specifically.

**Comment by nschonni** [comment]
It looks like this icon is in use in some places now, just not in the Notifications hub.
Feel free to close if there is nothing more for this repo

**Comment by yaili** [comment]
We will continue to use the pull request icon with different colors for variations.

**Comment by nschonni** [comment]
@yaili please re-open as this is still an accessibility issue with the GitHub site since they only leverage these color only difference in the icons

**Comment by nschonni** [comment]
Sorry, I realized that this one is a little outdated in the description, because this also affects the Draft vs. Open vs. Closed, but the Merged has been addressed now

**Comment by nschonni** [comment]
I've updated the description to remove the "Merged" icon, since that is now available

**Comment by nschonni** [comment]
Thanks for the Merged icon, it made my life a lot easier ❤️

**Comment by ashygee** [comment]
Hi @nschonni I'm going to add this to our Octicons V2 repo and we will keep track of it there. We definitely want our icons to be compliant with accessibility standards. Thanks for bringing this to our attention.
